nightwatch/features/event-reconciliation/reconciled.feature

```diff
@@ -0,0 +1,151 @@
+Feature: Reconciled
```

Comment:
Fine

On Wednesday, 4 May 2016, Chris Nesbitt-Smith <notifications@github.com>
wrote:

> In nightwatch/features/event-reconciliation/reconciled.feature
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6#discussion_r62001581>
> :
>
> > @@ -0,0 +1,151 @@
> > +Feature: Reconciled
>
> reconciled check in / check out events might be a better feature title?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6/files/8d568934e372574dcdb57ebb6c0def7d6fba6f57#r62001581>
>